versioned_docs/version-3.0.0-beta/framework-guide/deploy-hugo.md

```diff
@@ -0,0 +1,64 @@
+---
+id: deploy-hugo
+title: How to Deploy Hugo Apps in less than 30 Seconds
+sidebar_label: Deploy Hugo App
+slug: /framework-guild/deploy-hugo
+---
+
+This guide will show you how to deploy a Hugo project and set up your domain.
+
+[Hugo](https://gohugo.io/) is one of the most popular open-source static site generators. With its amazing speed and flexibility, Hugo makes building websites fun again.
+
+## Follow these steps to deploy your  Hugo app on Spheron successfully.
+
+### Step 1: Put Your Hugo Project Code in a Git Repo
+
+While this feature is not new to developers, the magic of Spheron is that every push to any branch in your GitHub, GitLab, or Bitbucket repo triggers a new build and deployment of your project.
+
+### Step 2: Update Your Hugo Project for Deploying on Decentralised Storage
```

Comment:
this section is not required for this. please remove it

Reply:
line 15 to 17 is actually required so people don't need to go and stat redeploying. if it is sgtep 2 that needs to be removed let me know

Reply:
yup line 15-17 is fine (I think github just showed it for no reason). But we don't need step 2 at all for Hugo deployment. Please remove it